cpp/src/io/parquet/page_data.cu

```diff
@@ -466,4 +468,33 @@ void __host__ DecodeSplitPageData(cudf::detail::hostdevice_span<PageInfo> pages,
   }
 }
 
+/**
+ * @copydoc cudf::io::parquet::detail::WriteOutputBufferSizesBatched
+ */
+void __host__ WriteFinalOffsetsBatched(std::vector<size_type> const& offsets,
```

Comment:
I think we can leave out __host__.
Don't know why we still have it on many functions.